README.md

```diff
@@ -1,7 +1,7 @@
 # KCron Common
 Cron realization for Kotlin Multiplatform
 
-[![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=Scogun_kcron-common&metric=alert_status)](https://sonarcloud.io/summary/new_code?id=Scogun_kcron-common) ![GitHub](https://img.shields.io/github/license/Scogun/kcron-common?color=blue) ![Publish workflow](https://github.com/Scogun/kcron-common/actions/workflows/publish.yml/badge.svg) [![Maven Central with version prefix filter](https://img.shields.io/maven-central/v/com.ucasoft.kcron/kcron-common/0.6.0?color=blue)](https://search.maven.org/artifact/com.ucasoft.kcron/kcron-common/0.6.0/jar)
```

Comment:
The library has `kcron-common` name because there are plans on next library, which will use this one as a base. I'm not sure that we should change name now.

Reply:
@Myshkouski will you revert that changes?